### PR TITLE
fix author noting bench

### PR DESCRIPTION
**Pull request by girazoki**
Fixes the author-noting benchmark that was not able to run properly for dancelight.

We also forgot to kill the author-data upon deregistration, something that this PR fixes too. It should not impact that much unless we re-register the same container, in which case we can remove this data manually

### CODE REVIEW

File: solo-chains/runtime/dancelight/src/lib.rs [hunk 1]
```diff
@@ -1737,9 +1737,6 @@ impl pallet_registrar::RegistrarHooks for DancelightRegistrarHooks {
     }
 
     fn para_deregistered(para_id: ParaId) -> Weight {
-        // Clear pallet_author_noting storage
-        // TODO: uncomment when pallets exists
-        /*
```
Comment (tmpolaczyk) [inline solo-chains/runtime/dancelight/src/lib.rs:1742]: Mention this change in the description

File: pallets/author-noting/src/benchmarks.rs [hunk 2]
```diff
@@ -55,26 +58,26 @@ mod test_sproof {
...
+                for para_id in 1..x {
+                    let para_id = para_id.into();
+                    container_chains.push(para_id);
+                    // Mock assigned authors for this para id
+                    let author: T::AccountId = account("account id", 0u32, 0u32);
+                    // Use the max allowed value for num_each_container_chain
+                    let num_each_container_chain = 2;
+                    T::ContainerChainAuthor::set_authors_for_para_id(para_id, vec![author; num_each_container_chain]);
+                    sproof_builder.num_items += 1;
+                }
```
Comment (tmpolaczyk) [inline pallets/author-noting/src/benchmarks.rs:78]: There's some whitespace here

**Review by tmpolaczyk — APPROVED** [review]

File: solo-chains/runtime/dancelight/src/lib.rs (hunk 1 above)
Comment (girazoki) [reply solo-chains/runtime/dancelight/src/lib.rs:1742]: yep

Comment (girazoki) [reply solo-chains/runtime/dancelight/src/lib.rs:1742]: fixed

File: pallets/author-noting/src/benchmarks.rs (hunk 2 above)
Comment (girazoki) [reply pallets/author-noting/src/benchmarks.rs]: fixed

**Pull request merged**